docs/CONTRIBUTING.md

```diff
@@ -168,7 +178,8 @@ setup properly.
 
 While not strictly required, we use [asdf-vm](https://asdf-vm.com) to manage
```

Comment:
```suggestion
We use [asdf-vm](https://asdf-vm.com) to manage
```
Nit: Could be shorter. None of this is strictly required since programmers usually wing it anyway and only read the instructions when we see an error